src/session.ts

```diff
@@ -755,26 +764,22 @@ export class SessionManager implements Middleware {
                 new SessionMenuItem(
                     "Restart Current Session",
                     () => { this.restartSession(); }),
+
+                ...powerShellItems,
             ];
+
         } else if (this.sessionStatus === SessionStatus.Failed) {
             menuItems = [
                 new SessionMenuItem(
                     `Session initialization failed, click here to show PowerShell extension logs`,
                     () => { vscode.commands.executeCommand("PowerShell.ShowLogs"); }),
+
+                new SessionMenuItem(
+                    "Restart Current Session",
+                    () => { this.restartSession(); }),
             ];
         }
 
-        const powerShellItems =
-            availablePowerShellExes
-                .filter((item) => item.exePath.toLowerCase() !== currentExePath)
-                .map((item) => {
-                    return new SessionMenuItem(
-                        `Switch to ${item.versionName}`,
-                        () => { this.changePowerShellExePath(item.exePath); });
-                });
-
-        menuItems = menuItems.concat(powerShellItems);
-
```

Comment:
I'd like to keep the `Switch to` options with a failed session if that's not too much trouble.  It's useful when testing features that may fail immediately in either Core or Full early on in the development process.  I used it a few times early on when working on PSRL before I figured out the module resolution issues.

Reply:
added back 😃